precompiles/staking/StakingI.sol

```diff
@@ -144,6 +145,18 @@ interface StakingI is authorization.AuthorizationI {
         uint256 value
     ) external returns (bool success);
 
+    /// @dev Defines a method for edit a validator.
+    /// @param description Description parameter to be updated, an empty string "" means keeping it as is and not updating it.
+    /// @param commissionRate CommissionRate parameter to be updated, -1 means keeping it as is and not updating it.
+    /// @param minSelfDelegation MinSelfDelegation parameter to be updated, -1 means keeping it as is and not updating it.
+    /// @return success Whether or not edit validator was successful
+    function editValidator(
+        Description calldata description,
+        string memory validatorAddress,
```

Comment:
use `address` here

Reply:
I disagree with your point.

Firstly, the validator addresses returned by the query interface are in the format of bench32, such as evmosvaloper1xxxx. If we use the address type, it would mean that users need to manually convert the bench32 format to address.

Secondly, in StakingI.sol, other functions such as `createValidator`, `delegate`, and `undelegate` all use string type for the validatorAddress input.

Of course, in the events, if we index the string type validator address, it stores the hashed validator address. In this case, we should change the string to address type for validator.

Reply:
we should only use address not bech32 moving forward

Reply:
If you confirm this, do you need to refactor the previous interface design to change all the validator to address type, and return the validator address in hex format instead of bech32?

Reply:
yes, that's the plan

Reply:
ok! understand.